miniwin/miniwin/include/miniwin_ddraw.h

```diff
@@ -267,7 +267,7 @@ struct IDirectDrawSurface3 : public IDirectDrawSurface {};
 typedef IDirectDrawSurface3* LPDIRECTDRAWSURFACE3;
 
 typedef HRESULT (*LPDDENUMMODESCALLBACK)(LPDDSURFACEDESC, LPVOID);
-struct IDirectDraw : public IUnknown {
+struct IDirectDraw : virtual public IUnknown {
```

Comment:
Can you apply this to all other IUnknown children?

Reply:
sure